### ISSUE TITLE
Have grouped feeds show a list of the channels you've put in them

### ISSUE THREAD

**Issue by grooku** [TeamNewPipe/NewPipe#3388]
#### Describe the feature you want

While the feeds are a very nice addition, I would also like to be able to see a list of the channels in the feed.

Alternatively, you could have a drop down menu on "Subscriptions" that allows you to narrow your subscriptions by feed type.

#### Is your feature request related to a problem? Please describe it

I want to a way to organize my subscription into groups.

#### Additional context

N/A

#### How will you/everyone benefit from this feature?

I was waiting for this grouping feature for a while!  While I like being able to see new videos of a feed group,  I was hoping for a way to organized my channels by genre. That was one of the main things I was hoping for - a way to organize my subscriptions. Being able to see the new videos is good, but it would be really nice to be able to have an organized list of the channels . 

In case the idea of having your subscription icons organized isn't enough, here are some possible uses of this idea: 
-Checking whether or not certain channels in your feed group have been inactive for months
-Being able to easily go on a binge watch of multiple channels older videos

Thank you!

**Comment by MD77MD** [comment]
#3353
New feed UI improved management #3353

Describe the feature you want
the ability to add subscribed channels to multiple existing feed groups directly from main subscription feed panel

Is your feature request related to a problem? Please describe it
Adding channels to a group require us to go through all subscribed channels.. which are alot, usually more than the feed groups

Additional context
My suggestion is to add an "add to group"
option within the -pressing and hold menu -
this option would open another drop-down menu containing all feed groups with a checkbox next to them. this is inspired from what youtube use to add videos to playlists

How will you/everyone benefit from this feature?
this would make adding and managing subscribed channels between feed groups a lot easier.

**Comment by MD77MD** [comment]
#3315
Older feed UI switch #3315

although the new feed UI has been a long awaited feature.. old feed UI definitely had its uses... combining both would be really awesome
I would suggest to add swipe left and right (like the one used between comments and related videos) to switch between two panels
-the first panel is used for listing feeds by upload time (new feed UI)
-the second panel will show latest video from each channel (old feed UI).. i would also like the ability added to go to any channel from here directly.. maybe by adding channel icon next to its related video or by an option in video menu.

if the swiping solution is not easy.. maybe a tab next to feed refresh bar would do the trick... or maybe make to tabs above the feed refresh bar, one called (new videos) and another called (channels).

also maybe you can make it an option to have it in settings... same as comments

**Comment by denemier** [comment]
yes please let us see channels from current feed on top or maybe as popup but when we click a channel than goto that channel

**Comment by opusforlife2** [comment]
I propose a solution in #3590.

**Comment by MD77MD** [comment]
> I propose a solution in #3590.

actually he is referring to something different then what your solution is for.

he is simply asking to the see the channels within feed group.

feed group=>channels list=>visit a channel

**Comment by opusforlife2** [comment]
@MD77MD
> Alternatively, you could have a drop down menu on "Subscriptions" that allows you to narrow your subscriptions by feed type.

I thought this part was similar enough to my proposal.

**Comment by SameenAhnaf** [comment]
@litetex Duplicate of #5592.

**Comment by SameenAhnaf** [comment]
In my opinion, the channel list should be shown in a single grid as suggested in https://github.com/TeamNewPipe/NewPipe/issues/3417. `+New` option should be first option so that users can add channels easily.

Short press on channel thumbnail: Show list of all groups with checkboxes. `+ New group`, `Notify` and `Unsubscribe` options should be added on top.
Short press on channel title: Go to channel feed

**Comment by MD77MD** [comment]
> In my opinion, the channel list should be shown in a single grid as suggested in #3417. `+New` option should be first option so that users can add channels easily.
> 
> Short press on channel thumbnail: Show list of all groups with checkboxes. `+ New group`, `Notify` and `Unsubscribe` options should be added on top. Short press on channel title: Go to channel feed

please see issue #3353 ... its almost identical to your idea... perhaps you can improve on it.

**Comment by opusforlife2** [comment]
For now, long-pressing is a workaround.

**Comment by bridgitiri** [comment]
I understand that long-pressing does technically allow you to see what channels you've selected to include in a group, but the only way to view these is as selected options within the entire list of every channel you are subscribed to. 

It is not very practical; and if you want to actually check the videos on a single channel you would need to enter the sub selection screen for the group, scroll around to find the channel, note the name if it's not one you frequently use/remember, return to the main screen, and scroll to find that channel or search to find it. 

It just adds a lot of steps to what could be: open group > see channel list > pick desired channel > see videos

**Comment by khimaros** [comment]
@SameenAhnaf @opusforlife2 i'm sorry, i must be dense. what are you long-pressing on in order to be able to choose the channel group to add a channel to? is this from the channel feed?

**Comment by khimaros** [comment]
FYI, the app [Grayjay](https://github.com/futo-org/grayjay-android) (a source-available, but not quite libre, alternative to NewPipe) can import NewPipe subscription JSON and supports this workflow for sorting subscriptions into groups.

i'd prefer to continue using NewPipe, but this workflow is absolutely critical to how i consume video content and is completely unusable with a large number of subscriptions in NewPipe's current implementation.

**Comment by khimaros** [comment]
ah, i see. that's the same workflow i've been using and it is completely impractical to use when you have 900+ subscriptions.

**Comment by MD77MD** [comment]
it's completely outdated... they should think out of the box like they guys at grayjay.
